integration-tests-ex/cases/pom.xml

```
@@ -346,7 +351,7 @@
                             </goals>
                             <configuration>
                                 <!-- our tests are very verbose, let's keep the volume down -->
-                                <redirectTestOutputToFile>true</redirectTestOutputToFile>
+                                <redirectTestOutputToFile>False</redirectTestOutputToFile>
```

Why is this change required? Also, if we go ahead with the change, then the comment above this should be updated to explain why it is set to false
```suggestion
                                <redirectTestOutputToFile>false</redirectTestOutputToFile>
```

---

Updated the comment to explain the change.